features/orders.feature

```diff
@@ -0,0 +1,91 @@
+Feature: The order service back-end
+    As an Order Service Manager
```

Comment:
Can we remove any tests related to the CRUD api's for now? We can add that when we implement them?